### PR TITLE
Create Documentation Enhancement Grants README.md

### PR DESCRIPTION
**Pull request by ErinOCon**

### CODE REVIEW

File: Program Resources/Documentation Enhancement Grants README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- For example, for applications submitted between March 1 and March 31, applicants will be notified by April 30.
+
+Our team may be in touch with questions in the proposal comment section in Github. **Please note that no work should begin without a signed contract.** 
+
+## Award
+Grants will be paid out once the following steps are completed:  
+- A grant agreement is signed by both parties
+- The outlined project work is complete
+  - As part of their final submission, all grantees should demonstrate successful usage by members of their intended audience with the intended audience as part of the project completion report.
```
Comment (realChainLife) [inline Program Resources/Documentation Enhancement Grants README.md:38]: Rephrase to "As part of the final submission"

File: Program Resources/Documentation Enhancement Grants README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      - For example, for a tutorial, it should be tested by three to five developers who are not part of the creation team.
+- The project work has been reviewed and approved by Filecoin Foundation
+- Once a completed project has been reviewed and approved, Filecoin Foundation will make a single payment of up to $5,000 USD or USDC within 30 days.
+
+## FAQs & Support
+For more information regarding our grant program, including other grant types, check out [Filecoin DevGrants](https://github.com/filecoin-project/devgrants). For specific questions, you can reach our team directly by:
+- Email at grants@fil.org
+- Slack at [#grants-help](https://filecoinproject.slack.com/archives/C02A48BJ1SQ)
+- Joining our [Office Hours](https://calendly.com/filecoin-grants/office-hours-ama?month=2024-05).
```
Comment (realChainLife) [inline Program Resources/Documentation Enhancement Grants README.md:47]: Edit the office hours like to remove month specification. Minor update to https://calendly.com/filecoin-grants/office-hours-ama

**Review by realChainLife — CHANGES_REQUESTED** [review]

File: Program Resources/Documentation Enhancement Grants README.md (hunk 1 above)
Comment (ErinOCon) [reply Program Resources/Documentation Enhancement Grants README.md:38]: Great catch@

File: Program Resources/Documentation Enhancement Grants README.md (hunk 2 above)
Comment (ErinOCon) [reply Program Resources/Documentation Enhancement Grants README.md]: Updated

**Pull request merged**